### ISSUE TITLE
Set opening direction?

### ISSUE THREAD

**Issue by T0miii** [charliekassel/vuejs-datepicker#347]
Is there a way to tell the datepicker in what direction it should open?
I encounter this problem:
![image](https://user-images.githubusercontent.com/10532141/32498949-6d323354-c3d1-11e7-95eb-a26b219821e3.png)

**Comment by charliekassel** [comment]
No... I used to measure the available space then open above or below but that messed up the SSR users.
Might be a nice addition to pass this in as a prop though...

**Comment by T0miii** [comment]
could i do this with the wrapper class?

**Comment by khflab** [comment]
This is a nice feature that I need just now

**Comment by alexandra1030** [comment]
wow, anyone has done this feature?

**Comment by nkwinder** [comment]
There is also a problem when the input is close to the right edge of the window... then only the half calendar is shown...
This is a big problem for responsive design

**Comment by IChris2698I** [comment]
I too would like this feature. I've got a datepicker near the bottom of the page, and it adds a scrollbar when opened. Top/right/bottom/left would be great :+1:

**Comment by briano-jabico** [comment]
I too would like this feature. I've got a datepicker near the bottom of the page, and it adds a scrollbar when opened. Top/right/bottom/left would be great :+1:

**Comment by robjbrain** [comment]
> No... I used to measure the available space then open above or below but that messed up the SSR users.
> Might be a nice addition to pass this in as a prop though...

@charliekassel i've tried to look in the commits to find where this code was removed and replicate it but I can't. Could you possibly point it out?

**Comment by joelhickok** [comment]
This is the **most rated** Vue Datepicker component that I have found so far, which is why I cannot believe that opening direction or position has not been resolved.  Component still opens off screen.  I will add it to my to do list, but it's not a high priority to me, since I'm pretty sure I can just find another component that has this built-in already.

![Screenshot 2019-05-07 09 58 07](https://user-images.githubusercontent.com/6742308/57314319-a799b680-70ae-11e9-87c3-c3386e78c862.png)

**Comment by sombatos** [comment]
Maybe add an option to display datepicker in the modal like https://mariomka.github.io/vue-datetime/

**Comment by HendraPB** [comment]
I also need this auto detect feature because I am currently experiencing the same problem, for now I solved it with css overide

.vdp-datepicker__calendar {
    width: 230px !important;
}

**Comment by hasibweb** [comment]
It's working, just use this css.
.vdp-datepicker__calendar {
  bottom: 100%;
}
